plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -389,7 +389,7 @@ static bool ffmpeg_hls_mux_start(void *data)
 	dstr_copy(&path, path_str);
 	dstr_replace(&path, "{stream_key}", stream_key);
 	dstr_init(&stream->muxer_settings);
-	dstr_catf(&stream->muxer_settings, "http_user_agent=libobs/%s",
+	dstr_catf(&stream->muxer_settings, "http_user_agent=libobs/%s method=PUT http_persistent=1",
```

Comment:
Make sure line length doesn't exceed 80 columns: https://screenshot.googleplex.com/KkTNmgG4CtV. Maybe run clang-format to make sure?